<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NextLevel</class>
 <widget class="QDialog" name="NextLevel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>594</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>594</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>19</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Congratulations! You WIN!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="restartButton">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>83</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/image/otherButton/res/image/otherButton/restartButton.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextButton">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>83</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/image/otherButton/res/image/otherButton/nextButton.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitButton">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>83</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/image/otherButton/res/image/otherButton/quitButton.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
